go.work

```diff
@@ -1,7 +1,7 @@
 go 1.22.1
 
-use ./bucket-blocker
-
 use ./ingress-inquisition
 
 use ./common
+
+use ./fsbp-fix
```

Comment:
after all the other projects have moved into this one, we can move it to the root